src/app/pluginmanager/qgspluginmanager.cpp

```diff
@@ -699,9 +699,6 @@ void QgsPluginManager::showPluginDetails( QStandardItem *item )
   }
 
   QString html = "<style>"
-                 "  body {"
-                 "    background-color:white;"
```

Comment:
Your fix works fine with non-webkit version of QgsWebView (built WITH_QTWEBKIT=False). 
However, in the webkit version (what is default I guess?), the default text color is always black and only the background is controlled by palette, what was apparently the reason for forcing the background to white. Your fix makes it black on black.

The best solution I can think out is forcing the color instead of background:

```
-                 "    background-color:white;"
+                 "    color:" + QPalette().color( QPalette::Text ).name() + ";"
```

Just tested with and without webkit on various color schemes and it seems to always work.

BTW. I've taken a look at other places when we use QgsWebView, but they aren't good examples to follow;) The Help>About >Translators defines a bright background for "body", but there is no `<body>` block in the page code. It surprisingly works fine, as Webkit seems to interpret the body as a global scope, while the non-webkit engine seems to only apply it to the actual `<body>` tag.

Reply:
Thanks for the review! Updated as suggested